### ISSUE TITLE
Train on Cub200-2011

### ISSUE THREAD

**Issue by FriedRonaldo** [xiaomengyc/ACoL#10]
Hi, I try training the model on CUB 200-2100.
To do this, I installed pytorch 0.4.1 and python 2.7 ( on a virtual environment of anaconda )

But it seems not to work well. 
I only changed the directory of the data from the .sh file.

The intermediate results are \:

Epoch: [12][160/300]	Time 0.412 (0.415)	ETA 01:40:33(00:02:04)	Loss 10.5779 (10.5851)	Prec@1 0.000 (0.440)	Prec@5 0.000 (2.264)
Epoch: [12][200/300]	Time 0.410 (0.414)	ETA 01:40:06(00:02:04)	Loss 10.5800 (10.5850)	Prec@1 0.000 (0.477)	Prec@5 0.000 (2.312)
Epoch: [12][240/300]	Time 0.410 (0.414)	ETA 01:39:42(00:02:04)	Loss 10.5862 (10.5848)	Prec@1 0.000 (0.460)	Prec@5 0.000 (2.176)
Epoch: [12][280/300]	Time 0.411 (0.413)	ETA 01:39:19(00:02:03)	Loss 10.5863 (10.5848)	Prec@1 5.000 (0.430)	Prec@5 5.000 (2.097)

The loss does not get better but stays at 10.5xxx

How can I solve this problem?
I also tried running the code on pytorch 1.0.0 and python 3.6 but, the result was the same as above.

Thanks :)

**Comment by kellenf** [comment]
I meet the problem too,maybe the hyper-param is not good enough,have you solved this problem??

**Comment by FriedRonaldo** [comment]
@kellenf No, I did not solve it.

**Comment by kellenf** [comment]
I have solve the problem,You need to change the lr->0.01 and  set a big epoch ,some code in this repo is wrong,need to do many change when use,good luck to you !

**Comment by FriedRonaldo** [comment]
@kellenf Good! Thanks for your comment.

**Comment by FuyaLuo** [comment]
@FriedRonaldo I think you might need change the file 'train_frame.py', in line 80, to make "pretrained=True", since the paper said they used pretrained models in ImageNet. By this mean, my training loss decreased normally.

**Comment by FriedRonaldo** [comment]
@FlorianLuo Thanks, I solved the problem in that way. :)
